pkg/server/container_stats.go

```diff
@@ -17,15 +17,34 @@ limitations under the License.
 package server
 
 import (
-	"errors"
+	"fmt"
 
+	tasks "github.com/containerd/containerd/api/services/tasks/v1"
 	"golang.org/x/net/context"
-
 	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
 // ContainerStats returns stats of the container. If the container does not
 // exist, the call returns an error.
 func (c *criContainerdService) ContainerStats(ctx context.Context, in *runtime.ContainerStatsRequest) (*runtime.ContainerStatsResponse, error) {
-	return nil, errors.New("not implemented")
+	// Validate the stats request
+	if in == nil || in.GetContainerId() == "" {
```

Comment:
will get rid of this

Reply:
nit: `in.GetContainerId() == ""` is enough